src/Storage/Device.php

```diff
@@ -271,9 +276,11 @@ abstract public function getPartitionTotalSpace(): float;
      * Get all files and directories inside a directory.
      *
      * @param  string  $dir Directory to scan
-     * @return string[]
+     * @param  int  $keys
+     * @param  string  $continuationToken
+     * @return array<mixed>
      */
-    abstract public function getFiles(string $dir): array;
+    abstract public function getFiles(string $dir, int $keys = self::MAX_KEYS, string $continuationToken = ''): array;
```

Comment:
Let's call this parameter $max instead of $keys

Reply:
Done 👍